packages/phone-number-privacy/common/src/test/values.ts

```diff
@@ -12,10 +11,13 @@ export const PRIVATE_KEY2 = '0x1234567890abcdef1234567890abcdef1234567890abcdef1
 export const ACCOUNT_ADDRESS2 = privateKeyToAddress(PRIVATE_KEY2)
 export const PRIVATE_KEY3 = '0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890fffff1d'
 export const ACCOUNT_ADDRESS3 = normalizeAddressWith0x(privateKeyToAddress(PRIVATE_KEY3))
-export const PHONE_NUMBER = '+15555555555'
+export const PHONE_NUMBER = '+17777777777'
```

Comment:
FYI this change breaks a bunch of the signer PNP/legacy integration tests (testing against fixed sigs that use the previous BLINDED_PHONE_NUMBER based off of +15555555555)